templates/search_results_layout_template.html

```diff
@@ -11,14 +11,25 @@
     <link rel="stylesheet" href="assets/css/styles.css">
     <!--Google Font Icon CSS-->
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/0ee567c03b.js" crossorigin="anonymous"></script>
 </head>
 
 <body>
     <div class="grid">
         <div class="header">
             <nav class="navbar navbar-light" style="background-color: #e3f2fd;">
                 <div class="container-fluid">
-                    <a class="nav-link" href="/v1/">New Search</a>
+                    <!--New Search-->
+                    <a class="nav-link" onclick="passLocation()" href="/v1/"><i class="fa-solid fa-magnifying-glass-location fa-2x" style='color:#5d9ba8'></i></a>
+                    <script>
```

Comment:
Did we miss the implementation of passLocation() here?

Reply:
What exactly does `onclick=passLocation()` do here?

Reply:
addressed in 3e3be036